Tests/test_NCBI_qblast.py

```diff
@@ -317,7 +317,9 @@ def run_qblast(
         if expected_hits is None:
             self.assertEqual(len(record.descriptions), 0)  # Expected no descriptions!
         else:
-            self.assertTrue(len(record.descriptions) > 0)  # Expected some descriptions!
+            self.assertGreater(
+                len(record.descriptions), 0
+            )  # Expected some descriptions!
```

Comment:
This might be clearer with the comment on the line above, and then the code will fit on one line (black style). What do you think?

Reply:
I let black do the formatting. It did so because of the inline-comment. But, your suggested style is better and will also maintain consistency with the above statement.